content/v2/openapi.yml

```diff
@@ -2776,6 +2781,27 @@ components:
       description: Only include results with a type field exactly matching the given string
       schema:
         type: string
+    FilterRegistrantChangeState:
+      name: state
+      in: query
+      required: false
+      description: Only include results with a state field exactly matching the given string
+      schema:
```

Comment:
Have you considered having this as an enum if the states are discrete?

Reply:
Thanks :+1: I did this.